llm_memory/service/memory_manager.py

```
@@ -155,6 +162,38 @@ def _is_scope_allowed(memory_scope: str, target_scope: str) -> bool:
             return scope == "public"
         return scope in {target, "public"}
 
+    def _apply_time_decay(self, memories: List[BaseMemory], decay_rate: float = 0.05) -> List[BaseMemory]:
+        """
+        对被动记忆应用时间衰减，并重新排序。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)